README.md

```diff
@@ -24,6 +24,12 @@ This project is organized into microservices. Although this repository contains
     * You will need to supply environmental variable to the microservices specifying each others' endpoints
```

Comment:
I think "environment variables" makes more sense here - environment vs. environmental, should also be plural since there are multiple endpoints.

Reply:
Thanks for the catch, I've made that mistake a couple times before. Done https://github.com/knative-portability/large-events/pull/39/commits/0027abd27173b1107cad33ce2aebba06ae606027